rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -19,9 +19,6 @@ info:
     name: GNU Lesser General Public License
     url: 'http://www.gnu.org/licenses/lgpl.html'
 
-servers:
-  - url: ${rest.server.url}
```

Comment:
❓I though this was here so that we could point the swagger UI to a real endpoint to test requests. Is that capability still available without these lines?

Reply:
That's the idea, but I'm not sure this ever gets resolved to the real URL.

Reply:
I read about this element online, but I didn't spend much time researching it. Near as I can tell, it is useful when you know the hostname of your production service. Anyone who wants to use the service knows where to send a request.  For example, the value "http://myservice.company.com/v2" might be what a requester would use to build a request to reach the live service.

![image](https://user-images.githubusercontent.com/133284/63127641-bde5c700-bf67-11e9-950f-e3def4fac47a.png)

Reply:
I don't quite know either, so I'm okay removing it. It's easy to add back in if we want it back. :+1: